packages/xo-web/src/xo-app/home/host-item.js

```diff
@@ -119,6 +122,14 @@ export default class HostItem extends Component {
                 <InconsistentHostTimeWarning host={host} />
                 &nbsp;
                 {hasLicenseRestrictions(host) && <LicenseWarning />}
+                &nbsp;
+                {(license === undefined || license.expires < Date.now()) && (
+                  <Tooltip content={_('hostNoSupport')}>
+                    <a href='https://xcp-ng.com' rel='noreferrer noopener' target='_blank'>
+                      <Icon icon='alarm' className='text-danger' />
+                    </a>
```

Comment:
Wasn't it supposed to open a modal first?

Reply:
Yes, but as discussed earlier, this PR will just add the icon + tooltip to the host's lvl, another PR will be created to replace the icons + tooltip with a modal.